src/Cli/ConfigGenerator.cs

```diff
@@ -221,7 +221,7 @@ public static bool TryCreateRuntimeConfig(InitOptions options, FileSystemRuntime
 
             if (runtimeBaseRoute is not null)
             {
-                if (!Enum.TryParse(options.AuthenticationProvider, ignoreCase: true, out EasyAuthType easyAuthMode) || easyAuthMode is not EasyAuthType.StaticWebApps)
+                if (!Enum.TryParse(options.AuthenticationProvider, ignoreCase: true, out AuthProvider easyAuthMode) || easyAuthMode is not AuthProvider.StaticWebApps)
```

Comment:
I think we should change the name of the variables to something like `authMode` if we are changing the enum type from `EasyAuthType` to `AuthType`